src/components/DeviceSelector.js

```diff
@@ -42,7 +34,9 @@ class DeviceSelector extends React.Component {
   render() {
     const { classes } = this.props;
     const { anchorEl } = this.state;
-
+    const devices = this.props.devices;
+    const activeDevice = this.getActiveDevice(devices);
+    const deviceName = activeDevice ? activeDevice.name : 'No device selected';
```

Comment:
The default values for `getActiveDevice` is an empty array, right? in what case `activeDevice` will be falsy?

Reply:
I think `getActiveDevice` should return an object (device) or null. In this case, you are returning an object (device) or an empty array (empty list of devices?)

However, this is just my opinion :p some people say `it's not a bug if it's documented` well... the same for functions

Reply:
I already fixed it, I got confused, that method should always return an object, it could be an empty object or the device object